### ISSUE TITLE
[Feature Request] Vanilla Damage to Elytra

### ISSUE THREAD

**Issue by Folas1337** [LuK050/ReElytras#1]
Currently when you set this config option it makes your Elytra not take any damage anymore:
```yaml
damagePerSecond: 0
```

Can we either make it so this option allows for vanilla damage on the elytra OR another config option which allows you to have vanilla damage behavior?

Because I like the option in the plugin that you can disable riptide and firework rockets but I'd like to keep enchantments like Unbreaking but they don't do anything since your plugin ticks down the durability every second, regardless of Unbreaking.

So in short: I'd love to see a way to enable vanilla damage behavior on the elytra rather than having a set amount of damage per second :)

**Comment by volychev** [comment]
Thanks a lot for finding the bug and using the plugin! I will definitely fix it as soon as possible

**Comment by Folas1337** [comment]
Oh and I just now realized that taking 1 damage every second is vanilla but Unbreaking isn't taken into account as I was saying.

So that's really the only issue 😅

**Comment by Folas1337** [comment]
As a fact... we just found out that wearing an Elytra while taking damage on any item/tool etc. will have Unbreaking ignored.

If I'm mining with an Elytra on and I have Unbreaking 3 on the tool, that'll be entirely ignored.
That's a pretty urgent problem honestly :sweat_smile:

**Comment by Folas1337** [comment]
I no longer need this
